plugins/markdown/Dockerfile

```diff
@@ -12,7 +12,12 @@ ADD . /python/src
 
 WORKDIR /python/src
 # Add any package dependencies here
-RUN apt-get update && apt-get install -y pandoc wkhtmltopdf --no-install-recommends
+RUN apt-get update && apt-get upgrade && apt-get install -y pandoc curl sudo --no-install-recommends
```

Comment:
do we need to install sudo and use `sudo apt-get clean`? and minor we could remove -v flag in curl

Reply:
Yep I need to fix this up I'll remove sudo where possible and take away the -v I had to git reset and lost some stuff